easybuild/easyconfigs/g/GULP/GULP-5.1-intel-2019a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'MakeCp'
+
+name = 'GULP'
+version = '5.1'
+
+homepage = 'https://gulp.curtin.edu.au/gulp/'
+description = """GULP is a program for performing a variety of types of simulation on materials
+ using boundary conditions of 0-D (molecules and clusters), 1-D (polymers), 2-D (surfaces, slabs
+ and grain boundaries), or 3-D (periodic solids)Band Unfolding code for Plane-wave based calculations"""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+toolchainopts = {'openmp': True}
+
+# download requires registration, https://gulp.curtin.edu.au/gulp/request.cfm?rel=download
+sources = [SOURCELOWER_TGZ]
+checksums = ['1acea105043495d94a3b4b19601dec8d48c3945a5036fd253eb2bf63c94150a2']
+
+dependencies = [
+    ('FFTW', '3.3.8'),
+]
+
+start_dir = 'Src'
+
+build_cmd = "./mkgulp -m -f -c intel"
+
+files_to_copy = [
+    (['gulp'], 'bin'),
+    (['../Docs', '../Examples', '../Libraries', '../Utils'], ''),
+]
+
+sanity_check_paths = {
+    'files': ['bin/gulp'],
+    'dirs': [],
```

Comment:
@migueldiascosta Maybe we should also list `Docs` & co here, to ensure they're in place as expected?